ubi.repo

```diff
@@ -0,0 +1,70 @@
+[ubi-8-baseos-rpms]
+name = Red Hat Universal Base Image 8 (RPMs) - BaseOS
```

Comment:
it is missing the python packages, probably because it was missed in the Containerfile.